transformers/storage/flip/initializers/knc_flip/initializer.go

```diff
@@ -0,0 +1,8 @@
+package knc_flip
```

Comment:
Do we need to also include a ZRX flip initializer?

Reply:
we already had one due to it existing in earlier Kovan deploys 🎉